data/pigui/views/map-sector-view.lua

```diff
@@ -17,6 +17,8 @@ local Color = _G.Color
 local ui = require 'pigui'
 local layout = require 'pigui.libs.window-layout'
 
+local Serializer = require 'Serializer'
```

Comment:
Teensy tiny nitpick but we generally prefer to require "core" modules like Serializer, Game, Event etc. at the top of the file before other requires. In this case, I'd move the `require 'Serializer'` statement to just after (or before, if you want to sort semi-alphabetically) `require 'SystemPath'`.